### PR TITLE
Localize '/docs/tasks/tools/install-minikube' path

### PR DESCRIPTION
**Pull request by mikonoid**
Added Minikube install documentation regarding that issue https://github.com/kubernetes-i18n-ukrainian/website/issues/14

### CODE REVIEW

**Comment by idvoretskyi** [comment]
@mikonoid please sign the CLA first as suggested above.

**Comment by sftim** [comment]
/check-cla

in case Prow is out of date

**Comment by sftim** [comment]
Ideally, _rebase this PR against the `master` branch_ first.

**Comment by sftim** [comment]
@mikonoid to make this preview OK you'll need to rebase it against _master_. Please also squash commits.

Would you like help with the rebase?

**Comment by mikonoid** [comment]
I will create new one pull request, because I made some mistakes with merge and I can't fix it.

**Comment by sftim** [comment]
You'll need to rebase this against _master_ then force-push your changes. Make sure you fetch _master_ from [k/website](https://github.com/kubernetes/website) before rebasing. After this rebase, this PR should show as 1 commit.

**Comment by sftim** [comment]
/hold

OK to cancel the hold if this PR has 1 commit and otherwise looks good to merge.

**Comment by mikonoid** [comment]
@sftim Can you help? Because I just tried to squash commits and rebase but got a new 16 commits in PR.

**Comment by Arhell** [comment]
> @sftim Can you help? Because I just tried to squash commits and rebase but got a new 16 commits in PR.

I usually do like you have 28 commits
git reset --soft HEAD ~ 28

then you make 1 commit and git push --force

**Comment by mikonoid** [comment]
@Arhell I did something wrong with squashing commits and PR was closed automatically.

**Comment by Arhell** [comment]
did you do synchronization?
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

**Comment by sftim** [comment]
OK, try this.
```
git checkout 6ea2d0f3bf309fc260a2a2553f39ea9883a49b98
cp content/uk/docs/tasks/tools/install-minikube.md /tmp/install-minikube.md
git checkout 'issue#14'
cp /tmp/install-minikube.md content/uk/docs/tasks/tools/install-minikube.md
git add content/uk/docs/tasks/tools/install-minikube.md
git commit -m 'Localize “Install Minikube” to Ukrainian' -m 'Add localized page.'
git push --force-with-lease
rm /tmp/install-minikube.md
```

**Comment by mikonoid** [comment]
yep, but

`fatal: reference is not a tree: 6ea2d0f3bf309fc260a2a2553f39ea9883a49b98`

I've cloned my fork anew

**Comment by mikonoid** [comment]
Ok, just pushed 1 commit, but tests are failed

**Comment by sftim** [comment]
You can look at the [deploy log](https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/5ef5fa2cc3b3190007ad5b5c) for a hint.

It doesn't look as this PR is rebased against _master_. Try this:
```
git checkout 'issue#14'
# next step only if you don't already have a remote for https://github.com/kubernetes/website
git remote add upstream https://github.com/kubernetes/website
# OK, back to task
git fetch upstream master
git rebase upstream/master
git push --force-with-lease
```

**Comment by mikonoid** [comment]
@sftim thanks! now is looking good

**Comment by sftim** [comment]
Preview: https://deploy-preview-22056--kubernetes-io-master-staging.netlify.app/uk/docs/tasks/tools/install-minikube/

/hold cancel

**Comment by MaxymVlasov** [comment]
@mikonoid please, add original as comments, as it specified in [Localizarion guidlines](https://kubernetes.io/uk/docs/contribute/localization_uk/)

File: content/uk/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+Перед тим, як використовувати `--driver=none`, ознайомтесь [з цієй документацієй](https://minikube.sigs.k8s.io/docs/reference/drivers/none/) для отримання додаткової інформації.
+{{< /caution >}}
+
+Minikube також підтримує `vm-driver=podman` схожий на Docker драйвер. Podman запущений як суперюзер (root user) це найкрайщий  шлях забезпечити повний доступ ваших контейнерів до будь-якої функції, наявної у вашій системі.
+
+{{< caution >}}
+`podman` драйвер вимагає запущені контейнери з під root користувача оскільки звичайні облікові записи користувачів не мають повного доступу до всіх функцій операційної системи, які, можливо, потребуватимуть їх роботи.
+{{< /caution >}}
+
+###Встановлення Minikube як Linux пакет
```
Comment (Arhell) [inline content/uk/docs/tasks/tools/install-minikube.md:97]: need indentation heer

File: content/uk/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -33,7 +37,7 @@ sysctl -a | grep -E --color 'machdep.cpu.features|VMX'
 
 {{% tab name="Windows" %}}
 Для перевірки, чи підтримується віртуалізація на  Windows 8 та версіях вище,запустіть наступну команду в терміналі
```
Comment (butuzov) [inline content/uk/docs/tasks/tools/install-minikube.md:39]:
```suggestion
Для перевірки, чи підтримується віртуалізація на  Windows 8 та версіях вище, запустіть наступну команду в терміналі
```

File: content/uk/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ card:
 
 <!-- overview -->
 
+<!--This page shows you how to install [Minikube](/docs/tutorials/hello-minikube),
+a tool that runs a single-node Kubernetes cluster in a virtual machine on your personal computer.
+-->
+
 Ця сторінка описує як встановити [Minikube](/docs/tutorials/hello-minikube), інструмент який дозволяє
 запустити  Kubernetes кластер з однієї ноди у віртуальній машині на персональному комп'ютері.
```
Comment (butuzov) [inline content/uk/docs/tasks/tools/install-minikube.md:17]: just nitpicking, it looks great but I would leave "your" in the translation. anyway, great work!

**Comment by butuzov** [comment]
Hi @sftim,  do you have any more concerns about the content?

**Comment by MaxymVlasov** [comment]
/remove-lifecycle stale
@sftim ping

**Comment by sftim** [comment]
> Hi @sftim, do you have any more concerns about the content?

I don't - feel free to approve, my earlier comments are all non-binding

**Pull request merged**